src/components/SchemaRow/SchemaRow.tsx

```diff
@@ -69,6 +69,25 @@ export const SchemaRow: React.FunctionComponent<SchemaRowProps> = React.memo(({
   const deprecated = isRegularNode(schemaNode) && schemaNode.deprecated;
   const validations = isRegularNode(schemaNode) ? schemaNode.validations : {};
   const hasProperties = useHasProperties({ required, deprecated, validations });
+  let isInternalSchemaError = false;
+  let internalSchemaError = 'You do not have permission to view this reference';
+
+  if ('fragment' in schemaNode) {
+    if ('x-sl-error-message' in schemaNode.fragment) {
+      isInternalSchemaError = true;
+      internalSchemaError = schemaNode.fragment['x-sl-error-message'] ?? internalSchemaError;
+    }
+    if (
+      'primaryType' in schemaNode &&
+      'items' in schemaNode.fragment &&
+      schemaNode.primaryType === SchemaNodeKind.Array &&
+      schemaNode.fragment.items !== null &&
+      'x-sl-error-message' in (schemaNode.fragment.items as any)
```

Comment:
is there away to avoid the `.items as any` twice in this block?

Reply:
Thanks Daniel for working with me on the type checking to ensure we didn't need to cast it as any!